test/Microsoft.AspNet.Localization.Tests/AccessLanguageHeaderRequestCultureProviderTest.cs

```diff
@@ -106,5 +106,38 @@ public class AccessLanguageHeaderRequestCultureProviderTest
                 var response = await client.GetAsync(string.Empty);
             }
         }
+        
+        [Fact]
+        public async void OmitDefaultRequestCultureShouldNotThrowNullReferenceException_And_ShouldGetTheRightCulture()
```

Comment:
How is this test different from `GetFallbackLanguage_ReturnsFirstNonNullCultureFromSupportedCultureList`. Except for the added `SupportedUICultures` list in this test, I don't see any difference.